ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+    post:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-deprovision
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      CREDENTIALS_MODE: Passthrough
+      OO_PACKAGE: gcp-filestore-csi-driver-operator
```

Comment:
This is for enabling CSI storage? Shall we reference cucushift-installer-rehearse-gcp-ipi-filestore-csi-workflow.yaml ?

Reply:
@jianli-wei if this is to enable cis storage, shall we append `chain: storage-conf-csi-optional-gcp-filestore` behind `cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision` chain directly, just like what `cucushift-installer-rehearse-gcp-ipi-filestore-csi-workflow.yaml` does ?

Reply:
or include `storage-conf-csi-optional-gcp-filestore` in `cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision` chain?

Reply:
@jianlinliu Let me clarify a bit, thanks! 

1. My workflow is only for userLabels & userTags testing. 
2. The chain `storage-conf-csi-optional-gcp-filestore` references several steps (see below), even only its 2nd step `storage-create-csi-gcp-filestore` would create something additional, see [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/storage/create/csi-gcp-filestore/storage-create-csi-gcp-filestore-commands.sh#L56-L87), which my testing doesn't want. 
```
chain:
  as: storage-conf-csi-optional-gcp-filestore
  steps:
    - ref: optional-operators-subscribe
    - ref: storage-create-csi-gcp-filestore
    - ref: storage-conf-wait-for-csi-driver
    - ref: storage-conf-storageclass-set-default-storageclass
```
3. That's why I didn't use the chain, and instead created a dedicated step for my testing.

Reply:
I am not sure if that might introduce some side effect, maybe @Phaow has more insight here.

From `cucushift-installer-check-gcp-user-labels-filestore-csi-commands.sh` and `cucushift-installer-check-gcp-user-labels-pd-csi-commands.sh`, seem like we are creating some storage resource in the post-check script, that means we also have to maintain the storage creation version by version, I has some concern that if we can always tracking all the potential storage part change by versions.

But if we leave that to `storage-conf-csi-optional-gcp-filestore` chain's owner, maybe we can save the effort.

Reply:
@jianlinliu @jianli-wei The [additional](https://github.com/openshift/release/blob/master/ci-operator/step-registry/storage/create/csi-gcp-filestore/storage-create-csi-gcp-filestore-commands.sh#L56-L87) create only used for our DEV's test chain, it should impact nothing. So reuse the exist chain will be better. Also for `cucushift-installer-check-gcp-user-labels-filestore-csi-commands.sh` and `cucushift-installer-check-gcp-user-labels-pd-csi-commands.sh` seems a bit complex(For such labels check we don't need to create pod, use Immediate storageClass should be ok), From my understanding, could we combine them as one ref or directly add to e2e test case(we could test in `gcp-ipi-compact-filestore-csi-f14` -> `gcp-ipi-compact-filestore-csi-user-tag-f14`, also reduce the duplicate  costs) will be better?